spec/features/admin/create_poem_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature 'Admin creates a new poem' do
+  scenario 'they see the poem on the index' do
+    visit '/admin/poems'
+
+    fill_in 'poem_greek_title', with: 'Titlos Alpha'
+    fill_in 'poem_english_title', with: 'Title A'
+    fill_in 'poem_greek_body', with: 'A Greek body is a wonderful thing.'
+    fill_in 'poem_english_body', with: 'An English body is less sought after.'
+    page.check('poem_published')
```

Comment:
The `check` method is mixed into RSpec, so you don't need to call it through the `page` object.

```rb
fill_in 'poem_english_body', with: '...'
check 'poem_published'
click_button 'Create Poem'
```